wgpu/src/lib.rs

```diff
@@ -1025,6 +1031,25 @@ impl<V: Default> Default for LoadOp<V> {
     }
 }
 
+/// Operation to perform to the output attachment at the end of a render pass.
+///
+/// Corresponds to [WebGPU `GPUStoreOp`](https://gpuweb.github.io/gpuweb/#enumdef-gpustoreop).
+#[derive(Copy, Clone, Debug, Hash, Eq, PartialEq, Default)]
+#[cfg_attr(feature = "trace", derive(serde::Serialize))]
+#[cfg_attr(feature = "replay", derive(serde::Deserialize))]
+pub enum StoreOp {
+    /// Stores the resulting value of the render pass for this attachment.
+    #[default]
+    Store,
+    /// Discards the resulting value of the render pass for this attachment.
+    ///
+    /// The attachment/affected texture aspect(s) will be treated as uninitialized afterwards.
```

Comment:
```suggestion
    /// The attachment's affected texture aspect(s) will be treated as uninitialized afterwards.
```

This.. might be clearer? Not a lot of people understand texture aspects, so I'm hesitant to write about it without something explaining what it is. Maybe a link back to the texture aspect type?

Reply:
we could also drop it from here, it's a super strange implementation detail about how different aspects of a texture can be in an uninitialized state (I hate it ;))

Reply:
It's not really an implementation detail, as it results in a visible clear happening - maybe we should document that behavior.

Reply:
Hm maybe just best to talk about depth/stencil aspects inline here, shouldn't be too hard, let me try.